hoverctl/cmd/export.go

```diff
@@ -22,10 +22,11 @@ will be written to the file path provided.
 
 		checkArgAndExit(args, "You have not provided a path to simulation", "export")
 
-		simulationData, err := wrapper.ExportSimulation(*target, urlPattern)
+		fsHelper := &configuration.LocalFileHelper{}
+		simulationData, err := wrapper.ExportSimulation(*target, fsHelper, urlPattern)
```

Comment:
I was wondering why you need to mock the file io, and realized you are writing the body file in `wrapper.ExportSimulation` which forces you to unit test it. 

Somehow it was decided that we called the HTTP client for hoverfly API, a "wrapper" in this project. It's wrapping the API resource, so file writing probably not belong to it, and should be outside. Then we can add some functional tests for the export command without any mocking.

Reply:
Oh. I got it.